pkg/kube/client.go

```diff
@@ -548,6 +552,18 @@ func (c *client) RunAndWait(stop <-chan struct{}) {
 	}
 }
 
+func (c *client) startInformer(stop <-chan struct{}) {
+	if !c.HasStarted() {
+		c.started.Store(true)
+		c.kubeInformer.Start(stop)
+		c.dynamicInformer.Start(stop)
+		c.metadataInformer.Start(stop)
+		c.istioInformer.Start(stop)
+		c.gatewayapiInformer.Start(stop)
+		c.extInformer.Start(stop)
+	}
```

Comment:
This is not right, say kubeInformer, it is a informer factory, we can use it to init pod/node/service, etc informers, if at one time only pod informer is initiated, and this function is called. But then node informer is initiated, and need to start, it will skip the start here.